course/staff_views.py

```diff
@@ -145,7 +148,43 @@ class EnrollStudentsView(CourseInstanceMixin, BaseFormView):
     form_class = EnrollStudentsForm
     template_name = "course/staff/enroll_students.html"
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["instance"] = self.instance
+        return kwargs
+
+    def enroll_from_sis(self) -> None:
```

Comment:
On a second thought, this should really be a CourseInstance method, and not here